l10n_be_mis_reports/data/new_mis_report_pl.xml

```diff
@@ -0,0 +1,398 @@
+            ref="mis_report_style_l10n_be_acc_det"
+        />
+    <field name="sequence">180</field>
+  </record>
+  <record model="mis.report.kpi" id="new_mis_report_pl_acc_66B">
+    <field name="report_id" ref="new_mis_report_pl" />
+    <field name="name">acc_66B</field>
+    <field name="description">Non-recurring financial charges [66B]</field>
+    <field name="expression">balp[66B%]</field>
+    <field name="type">num</field>
+    <field name="compare_method">pct</field>
+    <field name="style_id" ref="mis_report_style_l10n_be_3" />
+    <field name="auto_expand_accounts" eval="True" />
+    <field
+            name="auto_expand_accounts_style_id"
+            ref="mis_report_style_l10n_be_acc_det"
+        />
+    <field name="sequence">190</field>
+  </record>
+  <record model="mis.report.kpi" id="new_mis_report_pl_acc_9903">
+    <field name="report_id" ref="new_mis_report_pl" />
+    <field name="name">acc_9903</field>
+    <field name="description">Profit (Loss) for the period before taxes [9903]</field>
+    <field name="expression">+acc_9901 +acc_75_76B -acc_65_66B</field>
+    <field name="type">num</field>
+    <field name="compare_method">pct</field>
+    <field name="style_id" ref="mis_report_style_l10n_be_2i" />
+    <field name="sequence">200</field>
+  </record>
+  <record model="mis.report.kpi" id="new_mis_report_pl_acc_780">
+    <field name="report_id" ref="new_mis_report_pl" />
+    <field name="name">acc_780</field>
+    <field name="description">Transfer from deferred taxes [780]</field>
+    <field name="expression">balp[780%]</field>
+    <field name="type">num</field>
+    <field name="compare_method">pct</field>
+    <field name="style_id" ref="mis_report_style_l10n_be_2" />
+    <field name="auto_expand_accounts" eval="True" />
+    <field
+            name="auto_expand_accounts_style_id"
+            ref="mis_report_style_l10n_be_acc_det"
+        />
+    <field name="sequence">210</field>
+  </record>
+  <record model="mis.report.kpi" id="new_mis_report_pl_acc_680">
+    <field name="report_id" ref="new_mis_report_pl" />
+    <field name="name">acc_680</field>
+    <field name="description">Transfer to deferred taxes [680]</field>
+    <field name="expression">balp[680%]</field>
+    <field name="type">num</field>
+    <field name="compare_method">pct</field>
+    <field name="style_id" ref="mis_report_style_l10n_be_2" />
+    <field name="auto_expand_accounts" eval="True" />
+    <field
+            name="auto_expand_accounts_style_id"
+            ref="mis_report_style_l10n_be_acc_det"
+        />
+    <field name="sequence">220</field>
+  </record>
+  <record model="mis.report.kpi" id="new_mis_report_pl_acc_67_77">
+    <field name="report_id" ref="new_mis_report_pl" />
+    <field name="name">acc_67_77</field>
+    <field name="description">Income taxes on the result [67/77]</field>
+    <field name="expression">balp[67%,68%,69%,71%,72%,73%,74%,76%,77%]</field>
```

Comment:
This one is not correct. Looking at the official account chart, it's probably this:

```suggestion
    <field name="expression">balp[67%,77%]</field>
```